server/server.py

```diff
@@ -14,11 +14,12 @@ def index():
 def hello():
     text=''
     lang=''
-    if request.method == "POST":
+    if request.method == "GET":
```

Comment:
We send `GET` data through URL variables instead of `request.form` and `request.data` which are `POST` request specific. Flask has quick and easy way of handling them please read [documentation](https://flask.palletsprojects.com/en/1.1.x/quickstart/#variable-rules).

**Eample:**
- A `GET` endpoint that takes two variables 
```python
@app.route('/hello/<lang>/<text>')
def hello(lang, text):
  return f'{lang} {text}'
```

- Lets call it with Insomnia or the browser `http://127.0.0.1:5000/hello/en-us/something to say` the response should be `en-us something to say`

- please note that i'm not using `methods=['GET', 'POST']` since the default value for methods is `['GET']`

Reply:
Got it !!
It's working !

I will upload the full server folder changes in a minute.